tests/scripts/mbedtls_test.py

```diff
@@ -261,21 +261,26 @@ def test_vector_to_bytes(self, function_id, dependencies, parameters):
         data_bytes += bytearray([function_id, len(parameters)])
         for typ, param in parameters:
             if typ == 'int' or typ == 'exp':
-                i = int(param)
-                data_bytes += 'I' if typ == 'int' else 'E'
+                if re.match(r'(\d+|0x[0-9a-f]+)$',
```

Comment:
Don't rewrite the standard library: `i = int(param, 0)`

Reply:
Without this change, you will get the error:
~~~
`ValueError: invalid literal for int() with base 10:` that
is caused when a parameter is given in base 16. Use relevant base
when calling `int()` function.
~~~
It's either this change, or change the `generate_test_code.py` script to generate all the integers to base 10. ( some crypto tests have integers in base 16, for example [here](https://github.com/ARMmbed/mbed-crypto/blob/86268e1d302355ad8fd8e5ca5f1a7c7af8640678/tests/suites/test_suite_psa_crypto_init.data#L26))

Reply:
Just change `i = int(param)` to `i = int(param, 0)`.

Reply:
yes, thanks. I missed this part in the python documentation